vision-zero

```diff
@@ -249,6 +251,15 @@ def graphqlEngineUp(args):
         "docker-compose-ram-disk.yml" if args.ram_disk else "docker-compose-docker-volume.yml",
         "up", "-d", "graphql-engine"])
     subprocess.run(["docker", "ps"])
+    print("🚀  GraphQL Engine started.")
+    print("You can run './vision-zero hasura-console' to open the console.")
+
+def hasuraConsole(args):
+    print("🚀  Opening Hasura console")
+    try:
+        subprocess.call(["hasura", "console"], cwd="./atd-vzd")
+    except KeyboardInterrupt:
+        print('Exited Hasura console')
```

Comment:
Open to thoughts on this, my thinking was:
1. we run `./vision-zero replicate-db` to get the local db and Hasura running
2. it prompts what command to run next (like the Moped `hasura-cluster` does) to get the console running so we can generate schema changes and migrations through it
3. it was either `./vision-zero hasura-console` or `cd atd-vzd` and then `hasura console`
Also, I tried just starting the console as the last step of `./vision-zero replicate-db` but it errors if Hasura takes a moment to spin up and also locks up the terminal. Didn't want to go down that rabbit hole.